api/observability/v1/conditions.go

```diff
@@ -14,10 +14,6 @@ limitations under the License.
 
 package v1
 
-import (
```

Comment:
only the declaration of constants remained in this file

Reply:
Correct.  For now I'm certain these are still required.  The re-definitions follows the pattern recommended for API conditions and allows us to use the existing kube types